sound/soc/sof/ipc4-pcm.c

```diff
@@ -769,7 +769,7 @@ static void sof_ipc4_build_time_info(struct snd_sof_dev *sdev, struct snd_sof_pc
 					sdev->fw_info_box.offset;
 	sof_mailbox_read(sdev, info->llp_offset, &llp_slot, sizeof(llp_slot));
 	if (llp_slot.node_id != dai_copier->data.gtw_cfg.node_id) {
-		dev_info(sdev->dev, "no llp found, fall back to default HDA path");
+		dev_dbg(sdev->dev, "no llp found, fall back to default HDA path");
```

Comment:
I think I had the similar question on the original PR:
if the copier is HDA copier then LLP is not an option so there is no point of printing anything, right?
It is not something _any_ developer can do to switch from HDA codec to I2S/SDW without stellar soldering and board rework capabilities...

Reply:
@ujfalusi  so you prefer to remove this log ?

Reply:
well, if it is only printed for case where LLP would never been available, then yes.

Reply:
updated, thanks